LibAddonMenu-2.0/controls/panel.lua

```diff
@@ -48,7 +48,10 @@ local function ForceDefaults(panel)
 
     for i = 1, #panelControls do
         local updateControl = panelControls[i]
-        if updateControl.UpdateValue and updateControl.data.default ~= nil then
+        local updateControlData = updateControl.data
+        if updateControlData.resetFunc and type(updateControlData.resetFunc) == "function" then
```

Comment:
The first check is not needed, because checking for type "function" implicitly ensures that it exists.